packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -1485,6 +1485,29 @@ class Route extends EmberObject implements IRoute {
     return route && route.currentModel;
   }
 
+  /**
+    `_render` is used to render a template into a region of another template
+    (indicated by an `{{outlet}}`).
+
+    @method _render
+    @param {String} name the name of the template to render
+    @param {Object} [options] the options
+    @param {String} [options.into] the template to render into,
+                    referenced by name. Defaults to the parent template
+    @param {String} [options.outlet] the outlet inside `options.into` to render into.
+                    Defaults to 'main'
+    @param {String|Object} [options.controller] the controller to use for this template,
+                    referenced by name or as a controller instance. Defaults to the Route's paired controller
+    @param {Object} [options.model] the model object to set on `options.controller`.
+                    Defaults to the return value of the Route's model hook
+    @private
+   */
+  _render(name?: string, options?: PartialRenderOptions) {
```

Comment:
Refactors `render` to be `_render`, and call that from within `renderTemplate` and `render` instead.

Reply:
Let's use the `symbol` utility method from `@ember/-internals/utils` to make something more truly private.